fairscale/optim/oss.py

```diff
@@ -395,3 +424,64 @@ def _free_other_grads(self) -> None:
             for p in partition:
                 for t in p["params"]:
                     t.grad = None
+
+    @staticmethod
```

Comment:
This abstraction of _scatter doesn't seem necessary. I think the code would be simpler without it.

Reply:
it removed duplicated code in between oss and shardedddp, that was the only point basically. Without that the broadcast and the reduce code chunks are 50 lines each doing almost the same thing

Reply:
removed